src/main/kotlin/com/team4099/robot2022/Robot.kt

```diff
@@ -37,10 +41,18 @@ object Robot : TimedRobot() {
             { ControlBoard.turn.smoothDeadband(Constants.Joysticks.TURN_DEADBAND) })
 
     ControlBoard.resetGyro.whileActiveOnce(ResetGyroCommand())
+
+    val autonomousSelector = AutonomousSelector()
+  }
+
+  override fun robotInit() {
+    Drivetrain.zeroSteering()
+    Drivetrain.zeroGyro()
   }
 
   override fun autonomousInit() {
     // autonomousCommand.schedule()
+    autonomousSelector?.getCommand(Drivetrain)
```

Comment:
this should have `.schedule()` on the end of it so it actually schedules the autonomous command we get from the selector